services/monitoring/Makefile

```diff
@@ -173,23 +173,16 @@ smokeping_prober_config.yaml: smokeping_prober_config.yaml.j2 ${REPO_CONFIG_LOCA
 	$(call jinja, $<, $@);
 
 .PHONY: grafana/assets
-grafana/assets: .config.dir .shared_config.dir
-	$(eval CONFIG_DIR=$(shell cat $(REPO_BASE_DIR)/.config.dir))
-	$(eval SHARED_CONFIG_DIR=$(shell cat $(REPO_BASE_DIR)/.shared_config.dir))
-
-	$(eval GRAFANA_ASSETS_DIR=$(CONFIG_DIR)/assets/grafana)
-	@if [ ! -d $(GRAFANA_ASSETS_DIR) ]; then \
-		echo "Error: folder does not exist $(GRAFANA_ASSETS_DIR)" >&2; \
+grafana/assets: ${REPO_CONFIG_LOCATION}
+	@if [ ! -d "$(shell dirname ${REPO_CONFIG_LOCATION})/assets/grafana" ]; then \
```

Comment:
repo config location points to a file not directory.

Reply:
oh, there is a `dirname` here. 

Why not using `.config.dir` instead of `.config.location`? By doing so we won't need to use `dirname` everytime we need to set up a path

Reply:
Since dirname can always be used, it is not really all that relevant if we point to a file or directory in my eyes. It would be good to have a pre-commit script that checks all assumptions about the folder-structure, otherwise we will have issues with these deep-nested directory traversals in the future.... Anyway, doing it now would require changes everywhere, so in case we do it I would prefer it in a separate PR.